tests/client/amo/components/TestAddonDetail.js

```diff
@@ -61,6 +61,13 @@ function renderAsDOMNode(...args) {
 }
 
 describe('AddonDetail', () => {
+  const getClientCompatibilityFalse = () => ({
+    compatible: false,
+    maxVersion: null,
+    minVersion: null,
+    reason: INCOMPATIBLE_NOT_FIREFOX,
+  });
```

Comment:
It's a bit hard to keep return values like this for tests in sync with real code. Flow is really helpful here. I will nominate a Flow experiment for the sprint: https://github.com/mozilla/addons-frontend/issues/2044